core/src/main/java/io/github/tcdl/msb/adapters/mock/MockAdapter.java

```
@@ -50,9 +52,12 @@ public MockAdapter(String topic, Queue<ExecutorService> activeConsumerExecutors)
     public void publish(String jsonMessage) {
         LOG.debug("Received request {}", jsonMessage);
         try {
-            Message incomingMessage = Utils.fromJson(jsonMessage, Message.class);
-            pushRequestMessage(incomingMessage);
-        } catch (JsonConversionException e) {
+            JsonNode messageAsNode = jsonReader.read(jsonMessage);
+            if (messageAsNode.has("topics")) {
```

what if wit doesn't contain topics? Should we throw an exception?